k8s/vagrant/README.md

```diff
@@ -1,27 +1,36 @@
 # Installing Kubernetes Clusters on Ubuntu 16.04 using Vagrant
 
-OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. A deployer can choose the Vagrantfile, based on his choice of kubernetes version. 
+OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. 
 
-This Vagrantfile can be used on laptop or Baremetal server installed with Ubuntu 16.04 and Virtualization Enabled
+This Vagrantfile can be used on any machine with Virtualization Enabled, like laptop or baremetal server. 
 
+The below instructions will walk you through the following:
```

Comment:
Procedures listed in this section helps you -

Reply:
done